client/components/Tracker/Form/Form.jsx

```diff
@@ -1,30 +1,23 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import styled from 'styled-components';
-import { useUser } from '../../../context/UserContext';
 
-const Form = ({ getList }) => {
+const Form = ({ getList, renderCount, user }) => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
-  const user = useUser();
-  console.log('useContext -> ', user);
-
-
 
   const handleSubmit = (event) => {
```

Comment:
One comment- after a submitting, should the inputs be cleared?

Reply:
Will implement that feature in the near future.